vocabularies/qld-crs.ttl

```diff
@@ -9,90 +9,89 @@
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
 <http://linked.data.gov.au/def/queensland-crs> a owl:Ontology , skos:ConceptScheme ;

```

Comment:
There was surely a reason for the change, and I am sure that Nick would be the better person to speak to this, but I suspect there will be problems using notations for class names. 
Presumably we could have the same value for notation and preflabel, or at the least get the lookups populated from the notation in this vocab.

Reply:
> There was surely a reason for the change

Sorry but looking here I can't see what the change you're referring to is. Do you mean the merger of the Ontology & Concept Scheme (that's where the code line reference is at here).

`skos:notation` should really be used to indicate an ID for a concept from some other system, e.g. the Concept coded with some non-URI in a geology handbook or something. Can have multiple notations for multiple codes. 

I always like to use the primary code for the Concept in the URI if it is well known instead of a tokenised prefLabel but you can have both - the well-known code in the URI and in a `skos:notation`.